QgisModelBaker/gui/panel/pg_config_panel.py

```diff
@@ -440,3 +447,45 @@ def _keep_custom_settings(self):
             self.pg_ssl_mode_combo_box.currentData(),
             PgConfigPanel._SERVICE_COMBOBOX_ROLE.SSLMODE,
         )
+
+    def _fill_schema_combo_box(self):
+        connection = None
+        try:
+            connection = psycopg2.connect(
+                dbname=self.pg_database_line_edit.text(),
+                user=self.pg_auth_settings.username(),
+                password=self.pg_auth_settings.password(),
+                host=self.pg_host_line_edit.text(),
+                port=self.pg_port_line_edit.text(),
+            )
+
+        except OperationalError as exception:
+            logging.warning(f"Pg connection error: {exception}")
+            return
+
+        sql = """SELECT schema_name
+        FROM information_schema.schemata; """
+
+        cursor = connection.cursor()
+        cursor.execute(sql)
+
+        schemas = cursor.fetchall()
```

Comment:
Hm... Maybe this would be nicer to go over the db_connector like most requests we perform on the database. Or are there any limitations?

```
db_connector = db_utils.get_db_connector(configuration)
        if db_connector and db_connector.get_schema:     
```

Reply:
I thought about that but then I have seen that the pg_connector need a schema as parameter for the constructor.
But by re-looking at that I can just leave the schema empty... Will change this